model-server/src/main/kotlin/org/modelix/model/server/handlers/ContentExplorer.kt

```diff
@@ -138,8 +136,12 @@ class ContentExplorer(private val client: IModelClient, private val repoManager:
             div("nameField") {
                 attributes["data-nodeid"] = node.nodeId.toString()
                 b {
+                    val namePropertyUID = BuiltinLanguages.jetbrains_mps_lang_core.INamedConcept.name.getUID()
+                    val namedConceptRole = node.getPropertyRoles().find { it.endsWith(namePropertyUID) }
```

Comment:
Is there no more structured way to access this? Why is the `endsWith` required here?

Reply:
On second thought, it's not required.